C-Plus-Plus/README.md

```diff
@@ -281,6 +281,7 @@ _add list here_
 - [Two odd occuring](Bit--Manipulation/Two_odd_occuring.cpp)
 - [Power_Set_Using_Bitwise](Bit--Manipulation/Power_Set_Using_Bitwise.cpp)
 - [Swap_even_and_odd_bits](Bit--Manipulation/Swap_even_and_odd_bits.cpp)
+- [Find_duplicate_elements](Bit--Manipulation/Find_duplicate_elements.cpp)
```

Comment:
Remove `_` from [Find_duplicate_elements]

Reply:
so we need not to update readme with our code??

Reply:
We have to update